.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2
+jobs:
+  python-3.4:
+    docker:
+      - image: circleci/python:3.4
+
+    steps: &build
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "requirements.txt" }}
+
+      - run:
+          name: install-csslint
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install 8
+            nvm alias default node
+            npm install -g csslint
+
+      - run:
+          name: dependencies
+          command: pip install --user -r requirements.txt -r docs-requirements.txt -r test-requirements.txt
+
+      - run:
+          name: pytest
+          when: always
+          command: python3 -m pytest
+
+      - run:
+          name: coala-ci
+          when: always
+          command: |
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            pip install --user pip setuptools -U
+            python3 setup.py install
```

Comment:
this installs into /usr/...

Use `pip install --user .`

Reply:
So basically replace `python3 setup.py install` with `pip install --user` ?